projects/common/src/nop.spec.ts

```diff
@@ -0,0 +1,7 @@
+// this file can be removed as soon as there is a real test in the project
+
+describe('NopTest', () => {
+    it('should work if true is returned', () => {
+        expect(true).toBeTruthy();
+    });
+});
```

Comment:
remove?

Reply:
can't remove it, because the linter expects at least one test

Reply:
didn't you do the UUID test?

Reply:
or the tests from the pipes